core/templates/pages/collection-player-page/collection-footer/collection-footer.component.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
```

Comment:
Why do I see this file as a new file added? For other file changes I could this the file being renamed and then the corresponding changes.

Have you deleted that file and added this as a new file instead of renaming it?

Reply:
I accidentally forgot to add the untracked files while pushing, so I added it later.